include/IECoreImage/ImageReader.h

```diff
@@ -81,6 +82,9 @@ class IECOREIMAGE_API ImageReader : public IECore::Reader
 		/// If true, the values will not be linearized nor converted to float.
 		IECore::BoolParameter *rawChannelsParameter();
 		const IECore::BoolParameter *rawChannelsParameter() const;
+		/// The parameter specifying the miplevel to be read from the image file.
+		IECore::IntParameter *miplevelParameter();
```

Comment:
`mipLevel` perhaps? Camel case L?

Reply:
Addressed in dffb0de. Note to myself: This mixes ImageReader and TextureLoader, - needs fixing before squashing.